### ISSUE TITLE
Submitter's Form – Full screen view blocked by menu

### ISSUE THREAD

**Issue by LKchemposer** [responsible-ai-collaborative/aiid#1589]
When `Full screen` mode is activated, part of the window of text is blocked by the Menu on the side.

<img width="727" alt="image" src="https://user-images.githubusercontent.com/39468345/217208999-452d338e-a0fb-4b9a-978f-bd3a251a48ed.png">

Collapsing the Menu helps a bit, but the Menu still partially blocks the window

<img width="727" alt="image" src="https://user-images.githubusercontent.com/39468345/217209134-9b4607b1-9474-450d-b7cb-5a20793d4ed3.png">

**Comment by lmcnulty** [comment]
Closing in favor of #2164.
